### PR TITLE
LPS-65282 XSS Vulnerability in DDL

### PR DESCRIPTION
**Pull request by rafaprax**
//cc @marcellustavares

### CODE REVIEW

**Comment by brianchandotcom** [comment]
ci:stop

**Comment by brianchandotcom** [comment]
Merged. Thank you.
View total diff: https://github.com/brianchandotcom/liferay-portal/compare/8afda1f2c4...67140b3093

**Comment by neilking** [comment]
@samuelkong shouldn't this be fixed in taglib?

File: modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-web/src/main/resources/META-INF/resources/view_records.jsp [hunk 1]
```diff
@@ -87,7 +87,7 @@ recordSearchContainer.setOrderByType(ddlViewRecordsDisplayContext.getOrderByType
 <aui:nav-bar cssClass="collapse-basic-search" markupView="lexicon">
 	<c:if test="<%= ddlDisplayContext.isAdminPortlet() %>">
 		<aui:nav cssClass="navbar-nav">
-			<aui:nav-item label="<%= recordSet.getName(locale) %>" selected="<%= true %>" />
+			<aui:nav-item label="<%= HtmlUtil.escape(recordSet.getName(locale)) %>" selected="<%= true %>" />
```
Comment (samuelkong) [inline modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-web/src/main/resources/META-INF/resources/view_records.jsp]: @rafaprax @neilking 

Correct, this should probably be fixed in the taglib instead.

**Pull request closed without merging**